Framework/API/inc/MantidAPI/IndexProperty.h

```diff
@@ -33,7 +33,7 @@ namespace API {
   File change history is stored at: <https://github.com/mantidproject/mantid>
   Code Documentation is available at: <http://doxygen.mantidproject.org>
 */
-class MANTID_API_DLL IndexProperty : public Kernel::ArrayProperty<int> {
+class DLLExport IndexProperty : public Kernel::ArrayProperty<int> {
```

Comment:
That has been reverted. I did this because I noticed that other classes which inherited `PropertyValue` with some specific type only used the `DLLExport` macro instead of the general import/export macro.